org-formation/710-tgw/_tasks.yaml

```diff
@@ -142,6 +146,7 @@ Mappings:
       '055273631518': 'snowflakevpc'   # org-sagebase-scicomp
       '420786776710': 'bridge-aux'   # org-sagebase-bridgedev
       '325565585839': 'synapse-ops-vpc-v2' # org-sagebase-synapseprod
+      '797640923903': 'sagedefaultvpc' # org-sagebase-sageit
```

Comment:
TgwSpokeB is for secondary VPCs in the same account.  Since there are no VPCs in defined TgwSpokeA for sageit yet this should be moved to TgwSpokeA.

Reply:
I did not make the connection, will do.